### DISCUSSION TITLE
need more examples - not only spring boot

### DISCUSSION THREAD

**Discussion by guykatz** [graphql-java/graphql-java#2876] [General]
many people are using non spring based frameworks.
the docs examples here are spring-boot heavy this doesnt really help the cause...
my 2 cents :)
thanks

**Comment by dondonz** [comment]
We always welcome more documentation contributions. If you or others reading this would like to write guides for non-Spring based frameworks, feel free to open a PR on https://github.com/graphql-java/graphql-java-page
